budgetportal/fixtures/provincial-infrastructure-projects.json

```diff
@@ -1,6 +1,6 @@
 [
 {
-  "model": "budgetportal.provinfraproject",
+  "model": "budgetportal.provinfraprojectsnapshot",
```

Comment:
Let me please know if it does make sense, I think this fixture is a bit outdated but maybe I'm wrong.

Reply:
is any test using it? how does this even work?

Reply:
I do not think so, I just used it as part of my initial loading data tries (before you sent those files to me)

Reply:
I checked this - its not being used.

Reply:
Thanks - I added it to the dev setup docs.